src/query/ee/src/test_kits/mock_licnese_mgr.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2023 Databend Cloud
+//
+// Licensed under the Elastic License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.elastic.co/licensing/elastic-license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use common_base::base::GlobalInstance;
+use common_exception::exception::ErrorCode;
+use common_exception::Result;
+use common_exception::ToErrorCode;
+use common_license::license::LicenseInfo;
+use common_license::license_manager::LicenseManager;
+use common_license::license_manager::LicenseManagerWrapper;
+use common_settings::Settings;
+use dashmap::DashMap;
+use jwt_simple::algorithms::ECDSAP256KeyPairLike;
+use jwt_simple::claims::JWTClaims;
+use jwt_simple::prelude::Claims;
+use jwt_simple::prelude::Clock;
+use jwt_simple::prelude::Duration;
+use jwt_simple::prelude::ECDSAP256PublicKeyLike;
+use jwt_simple::prelude::ES256KeyPair;
+
+fn build_custom_claims(license_type: String, org: String) -> LicenseInfo {
+    LicenseInfo {
+        r#type: Some(license_type),
+        org: Some(org),
+        tenants: Some(vec!["test".to_string()]),
+    }
+}
+
+pub struct MockLicenseManager {
+    // cache available settings to get avoid of unneeded license parsing time.
+    pub(crate) cache: DashMap<String, JWTClaims<LicenseInfo>>,
+}
+
+impl LicenseManager for MockLicenseManager {
```

Comment:
I don't think we need additional MockLicenseManager, since we could use RealLicenseManager::new(public_key) for customizable test suites

Reply:
ok